2020-05-06 22:22:00,969 ERROR: Exception on /user/Maksks [GET] [in /Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/yanalazareva/Documents/myshop/myshop/app/routes.py", line 119, in user
    posts = user.posts.order_by(Post.timestamp.desc()).paginate(
NameError: name 'Post' is not defined
2020-05-06 22:22:01,983 ERROR: Exception on /user/Maksks [GET] [in /Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/yanalazareva/Documents/myshop/myshop/app/routes.py", line 119, in user
    posts = user.posts.order_by(Post.timestamp.desc()).paginate(
NameError: name 'Post' is not defined
2020-05-06 22:22:07,833 ERROR: Exception on /user/Maksks [GET] [in /Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/yanalazareva/Documents/myshop/myshop/app/routes.py", line 119, in user
    posts = user.posts.order_by(Post.timestamp.desc()).paginate(
NameError: name 'Post' is not defined
2020-05-06 22:22:12,961 ERROR: Exception on /user/Maksks [GET] [in /Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/yanalazareva/Documents/myshop/myshop/app/routes.py", line 119, in user
    posts = user.posts.order_by(Post.timestamp.desc()).paginate(
NameError: name 'Post' is not defined
2020-05-06 22:22:17,245 ERROR: Exception on /user/Maksks [GET] [in /Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/yanalazareva/Documents/mycinema/myvenv/lib/python3.7/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/Users/yanalazareva/Documents/myshop/myshop/app/routes.py", line 119, in user
    posts = user.posts.order_by(Post.timestamp.desc()).paginate(
NameError: name 'Post' is not defined
2020-05-06 22:27:55,081 INFO: Shop startup [in /Users/yanalazareva/Documents/myshop/myshop/app/__init__.py:48]
2020-05-06 23:21:05,560 INFO: Shop startup [in /Users/yanalazareva/Documents/myshop/myshop/app/__init__.py:48]
2020-05-07 09:01:03,195 INFO: Shop startup [in /Users/yanalazareva/Documents/myshop/myshop/app/__init__.py:48]
